code/lib/cli/src/automigrate/fixes/wrap-require.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-param-reassign */
+import chalk from 'chalk';
+import { dedent } from 'ts-dedent';
+import * as t from '@babel/types';
+import type { Fix } from '../types';
+import { detectPnp } from '../../detect';
+import { updateMainConfig } from '../helpers/mainConfigFile';
+
+interface WrapRequireRunOptions {
+  storybookVersion: string;
+}
+
+export const wrapRequire: Fix<WrapRequireRunOptions> = {
+  id: 'wrap-require',
+
+  async check({ packageManager, storybookVersion }) {
+    const isStorybookInMonorepo = await packageManager.isStorybookInMonorepo();
```

Comment:
What if the content of the main.js file contains wrapForPnp already? I guess the chances are too low?

Reply:
This case is caught in lines `56` and `67`. If the value of `framework`, `builder`, ... is not a string, it will not be replaced. Still, the prompt would appear, which is not pleasant. I will extract a helper function to cover this case and to avoid the automigration running.